query-engine/prisma-models/src/fields.rs

```diff
@@ -1,26 +1,17 @@
 use crate::pk::PrimaryKey;
 use crate::*;
-use once_cell::sync::OnceCell;
-use std::{collections::BTreeSet, sync::Arc};
+use psl::parser_database::ScalarFieldType;
+use std::collections::BTreeSet;
 
 #[derive(Debug, Clone)]
 pub struct Fields {
```

Comment:
What is the purpose of this struct?

Reply:
:shrug: 

It exists, so I'm not deleting it, but after today's next 2 PRs, it will probably a good target for a morning PR to remove that and absorb the methods back into Model.